module UserAlreadyExistsException;

UserAlreadyExistsException::UserAlreadyExistsException() :
    std::logic_error("")
{}